keyboards/keebwerk/nano_slider/keebwerk_nano_via.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
This file does not belong in this repo and should be removed.

Reply:
New commit made to remove the JSON file